src/components/CalibrationManagerCells.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalCloseButton,
+  Button,
+  Th,
+  Td,
+  Link,
+} from "@chakra-ui/react";
+import trpc from "trpc";
+import ModalWithBackdrop from "./ModalWithBackdrop";
+import UserSelector from "./UserSelector";
+import { useState } from "react";
+import { formatUserName } from "shared/strings";
+import { MdEdit } from 'react-icons/md';
+import { Calibration as SharedCalibration } from 'shared/Calibration';
+
+export function CalibrationManagerHeaderCells() {
+  return <>
+    <Th>面试主管</Th>
+  </>;
+}
+
+export function CalibrationManagerCells({ calibration, refetch }: {
+  calibration: SharedCalibration,
+  refetch: () => void,
+}) {
+  const [editing, setEditing] = useState<boolean>(false);
+  return <>
+    {editing && <ManagerEditor calibration={calibration} 
+      onClose={() => setEditing(false)} refetch={refetch} />}
+
+    <Td><Link onClick={() => setEditing(true)}>
+      {calibration.manager ?
+        formatUserName(calibration.manager.name)
+        :
+        <MdEdit />
+      }
+    </Link></Td>
+  </>;
+}
+
+function ManagerEditor({ calibration, refetch, onClose } : {
+  calibration: SharedCalibration,
+  refetch: () => void,
+  onClose: () => void,
+ }) {
+  const saveManager = async (managerIds: string[]) => {
+    if (managerIds.length == 0) return;
```

Comment:
previous comment on this line isn't addressed

Reply:
See my reply: https://github.com/yuanjian-org/app/pull/416#discussion_r1803906402
Do you still want to change to:
`if (managerIds.length == 0) return null;`